wrangler-core/src/main/java/io/cdap/wrangler/parser/RecipeVisitor.java

```diff
@@ -310,7 +310,8 @@ public RecipeSymbol getCompiledUnit() {
     List<TerminalNode> strings = ctx.String();
     List<String> strs = new ArrayList<>();
     for (TerminalNode string : strings) {
-      strs.add(string.getText());
+      String text = string.getText();
+      strs.add(text.substring(1, text.length() - 1));
```

Comment:
do we always expect the text to be in the format of `"text"`?

Reply:
What’s the contract of `string.getText()`? Will it ever return an empty string?

Reply:
Yes, the contract is defined by the grammar in Directive.g4 (https://github.com/data-integrations/wrangler/blob/develop/wrangler-core/src/main/antlr4/io/cdap/wrangler/parser/Directives.g4#L272)

Reply:
if it can return an empty string, it needs to be checked before the substring call

Reply:
a text without '' or "" will never trigger this. So atleast there will be two characters present. That is defined by the grammar. The state machine wouldn't get you here.